File: 8-kyu/java/how-old-will-i-be-in-2099/how-old-will-i-be-in-2099.java
public class AgeDiff {
  public static String CalculateAge(int birth, int yearTo) {
    int date = Math.abs(yearTo - birth);
    if (date == 0) {return "You were born this very year!";}
    if (yearTo > birth) {
      if (date == 1) {
        return String.format("You are %d year old.", date);
      }
      return String.format("You are %d years old.", date);
    }
    else {
      if (date == 1) {
        return String.format("You will be born in %d year.", date);
      }
      return String.format("You will be born in %d years.", date);     
    }
  }
}

File: 8-kyu/python/how-old-will-i-be-in-2099/how-old-will-i-be-in-2099.py
def calculate_age(year_of_birth, current_year):
    date = abs(current_year - year_of_birth)
    if date == 0:
            return f"You were born this very year!"
    if current_year >= year_of_birth:
        if date == 1:
            return f"You are {date} year old."
        return  f"You are {date} years old."
    else:
        if date == 1:
            return f"You will be born in {date} year."
        return f"You will be born in {date} years."